composer.json

```diff
@@ -27,14 +27,15 @@
         "php": "^5.6 || ^7.0",
         "symfony/framework-bundle": "^2.7 || ^3.0",
         "cache/taggable-cache": "^0.5",
-        "cache/session-handler": "^0.2"
+        "cache/session-handler": "^0.2",
```

Comment:
It was not a part of this PR. 

But here: https://github.com/php-cache/cache-bundle/pull/76